todo/tests/integration/__init__.py

```diff
@@ -0,0 +1 @@
+# Added this because without this file Django isn't able to auto detect the test files
```

Comment:
This file is used to make the package identified as python module.
If we remove it we cannot import the classes / functions outside in other classes.